### ISSUE TITLE
🎨 Entity Info Text Improved Formatting

### ISSUE THREAD

**Issue by Felix-The-Ghost** [assaultcube/AC#287]
I would like some features from [my 1.2 mapping client](https://forum.cubers.net/thread-7663.html) to be considered for inclusion in the upcoming 1.3 release.

While there are many features to consider this issue will focus on the entity info present in edit mode.

Description of proposed features from forum post:

> Entity sparkles are now colorized according to the corresponding entity so identifying them from a distance is easier (arrows from playerstart entities also colorized)

> The closest/selected entity now radiates smoke particles while retaining its sparkle color.

> The entity information text that is visible on the bottom-left of the screen has been reformatted into three lines: the first shows the selection status of entities, the second is the standard entity info (entity name is now colorized to match the entity sparkle), and the third line displays the file name/location of the resource on disk for sounds and mapmodels.

In-game appearance:

![sparkles](https://user-images.githubusercontent.com/7077936/116606038-c4f86880-a8f5-11eb-8c04-8bfdcb54234f.gif)

Due to hardware failure I have no personal machine to actually make a pull request or even check out 1.3 but I can show the [diff from 1.2 source](https://pastebin.com/5eyEYbx8) and point out the following sections if assimilation is considered:

editinfo section - line 143

entcolor section - line 101

renderentities section - line 1639

worldrender section - line 1796

MAXPARTYPES - line 1880

particle struct - line 1883

**Comment by BrutalSystem** [comment]
> Entity sparkles are now colorized according to the corresponding entity so identifying them from a distance is easier (arrows from playerstart entities also colorized)

I prefer equal to sauerbraten
![2021-05-01_22 21 57_abyss_coop-edit](https://user-images.githubusercontent.com/7694535/116799100-5d484600-aacc-11eb-8d60-2105820b19ae.png)

**Comment by Felix-The-Ghost** [comment]
> I prefer equal to sauerbraten

Would you mind elaborating what you mean by this? I see the screenshot but I'm unable to infer from your brief message if you simply mean all sparkles being blue or if you want a verbose amount of information on the HUD, or those rectangles around the entities, or labels directly on the entities.

If the only issue with this proposal is preference, it can always be programmed as an option (e.g. "legacy entity colors")

**Comment by BrutalSystem** [comment]
> > I prefer equal to sauerbraten
> 
> Would you mind elaborating what you mean by this? I see the screenshot but I'm unable to infer from your brief message if you simply mean all sparkles being blue or if you want a verbose amount of information on the HUD, or those rectangles around the entities, or labels directly on the entities.
> 
> If the only issue with this proposal is preference, it can always be programmed as an option (e.g. "legacy entity colors")

you are right, which I say "I prefer equal to sauerbraten" I mean that on top of the entity particle it should have the name of the entity
